Framework/Algorithms/test/MaxEntTest.h

```diff
@@ -1073,6 +1132,35 @@ class MaxEntTest : public CxxTest::TestSuite {
     TS_ASSERT_EQUALS(data->readX(0), ws->readX(0));
   }
 
+  MatrixWorkspace_sptr createWorkspaceWithYValues(size_t nHist, size_t length, std::vector<double> const &YVal) {
+
+    size_t nPts = length * nHist;
+    TS_ASSERT_EQUALS(nPts, YVal.size());
+    MantidVec X(nPts);
+    MantidVec Y(nPts);
+    MantidVec E(nPts);
```

Comment:
could we please use the new histogram data types

Reply:
This is covered by issue #23106